### PR TITLE
EZP-29716: It is not possible to translate Admin UI footer

### PR DESCRIPTION
**Pull request by bdunogier**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29716
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Makes the copyright text in the admin base template translatable.

#### Checklist:
- [x] Ready for Code Review
- [ ] Synchronize translations once merged

### CODE REVIEW

File: src/bundle/Resources/views/Security/base.html.twig [hunk 1]
```diff
@@ -25,7 +25,13 @@
                         {%- endblock content -%}
                     </div>
                     <div class="ez-login__footer">
-                        <span>eZ Platform ©1999-{{ "now"|date("Y") }} <a href="http://ez.no">eZ Systems AS</a> and others.</span>
+                        <span>{{ ''|trans({
```
Comment (adamwojs) [inline src/bundle/Resources/views/Security/base.html.twig]: Translation key is missing 😉

Comment (bdunogier) [reply src/bundle/Resources/views/Security/base.html.twig]: 👼 focused too much on the link, and I got sidetracked. Thank you ^^

Comment (bdunogier) [reply src/bundle/Resources/views/Security/base.html.twig]: Should be fixed now. Not 100% sure about the key, let me know if you have a better one.

File: src/bundle/Resources/translations/messages.en.xliff [hunk 2]
```diff
@@ -6,6 +6,11 @@
       <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
     </header>
     <body>
+      <trans-unit id="da39a3ee5e6b4b0d3255bfef95601890afd80709" resname="base.footer_copyright">
+        <source>eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</source>
+        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</target>
+        <note>key: </note>
```
Comment (dew326) [inline src/bundle/Resources/translations/messages.en.xliff]: Don't know if it matters but the key name is missing here.

Comment (bdunogier) [reply src/bundle/Resources/translations/messages.en.xliff]: Oh, thank you. Regenerating the new string didn't work, and I ended up updating it manually. I'll add it.

Comment (bdunogier) [reply src/bundle/Resources/translations/messages.en.xliff]: Added the missing key, fixed up and rebased.

**Comment by barbaragr** [comment]
Testing is blocked by https://jira.ez.no/browse/EZP-29726

**Comment by bdunogier** [comment]
@barbaragr does this happen without this branch ? I'm kind of assuming that it does...

**Comment by barbaragr** [comment]
@bdunogier Yes, it happens without this branch, so I'm blocked with testing this PR.

**Comment by lserwatka** [comment]
@bdunogier do you have any progress with this issue?

**Comment by bdunogier** [comment]
Back a while ago, I did confirm the issue, and explained what settings I had to force to get translations to show.

It confirms that a) this PR is valid, as the string can be translated b) there is a deeper issue with the translation locale in the admin. I tried to look into it, but didn't really find anything relevant. I wanted to ask a a core dev back then, but they were too busy or on vacation.

Please try to find somebody to look into what's described in [this comment](https://jira.ez.no/browse/EZP-29726?focusedCommentId=233156&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-233156).

**Comment by SylvainGuittard** [comment]
@Plopix reported an issue regarding the `Symfony locale`: https://jira.ez.no/browse/EZP-29902.
It might be related.

**Comment by lserwatka** [comment]
Fixed as part of https://github.com/ezsystems/ezplatform-admin-ui/pull/756

**Comment by Plopix** [comment]
reported on Friday fixed on Monday ;-) pretty neat!

**Pull request merged**